R/dummy_functions.R

```diff
@@ -9,6 +9,9 @@
 #'   data = teal_data(IRIS = iris, MTCARS = mtcars),
 #'   modules = example_module()
 #' )
+#' \dontshow{
```

Comment:
Do we need this here ?

Reply:
sorry we dont! I though I removed that

Reply:
removed with 2a9ced574473ded8e5569eaec98299163c7540f6